src/index.ts

```diff
@@ -20,11 +20,18 @@ const client = new CookiecordClient(
 		prefix: ['!', 't!'],
 	},
 	{
-		ws: { intents: Intents.NON_PRIVILEGED },
 		partials: ['REACTION', 'MESSAGE', 'USER', 'CHANNEL'],
-		disableMentions: 'everyone',
+		allowedMentions: {
+			parse: ['users'],
+		},
+		intents: new Intents([
+			'GUILDS',
+			'GUILD_MESSAGES',
+			'GUILD_MEMBERS',
+			'GUILD_MESSAGE_REACTIONS',
+		]),
```

Comment:
I *think* these are all of the intents that we need

Reply:
In DMs:
```
ckie — Today at 6:57 PM
t!help
TypeScript BOT — Today at 6:57 PM
:warning: command was inhibited: not in a guild
```
Might also want the intents for DMs for this very minor case.